.github/workflows/codeql.yml

```diff
@@ -0,0 +1,46 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+      - master
+      - branch/*
```

Comment:
The branch names on this repository are different from `teleport`. It's actually `teleport-v*`.